docs/markdown/android/textarea.md

```diff
@@ -0,0 +1,92 @@
+
+
+### Comprehension
+e sure to [provide instructions](https://www.w3.org/WAI/tutorials/forms/instructions/) to help users understand how to complete the form and use individual form controls.
+
+### Labels
+Ensure the labels are precise and concise. Labels should only describe the text field they are associate with, and they must be visible.
+
+
+### Validation
+When providing a validation message, make sure the instructions are clear and help users complete the field. For example, "This field is required to submit". In addition, use the helper text to provide instructions to help users understand how to complete the text field or to indicate any needed input, allowed formats, timing limitations, or other pertinent information.
+
+These practices give users of assistive technologies more information about the form, helping them to fill it out.
+
+
+## Variants
+
+### Default
+TextArea will expand to fill the width of the parent container by default.
+<iframe style={{border:0}} width="100%" height="300" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FREw1COFYAktmVWrUBh3Ov8%2Fbranch%2FTxlMtK2AcYHeTliywyAyU7%2FGestalt-for-Android%3Fnode-id%3D12157%253A22670%26t%3DKJZySMQdq4Hp0RLc-1" allowFullScreen></iframe>
+
+### Helper text
+Whenever you want to provide more information about a form field, you should use helperText.
+<iframe style={{border:0}} width="100%" height="300" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FREw1COFYAktmVWrUBh3Ov8%2Fbranch%2FTxlMtK2AcYHeTliywyAyU7%2FGestalt-for-Android%3Fnode-id%3D12157%253A22816%26t%3DKJZySMQdq4Hp0RLc-1" allowFullScreen></iframe>
+
+### Disabled
+TextArea can be disabled to indicate the user is unable to interact with it. Disabled fields do not need to pass contrast requirements, so do not use a disabled TextArea to present information important information to the user.
+<iframe style={{border:0}} width="100%" height="300" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FREw1COFYAktmVWrUBh3Ov8%2Fbranch%2FTxlMtK2AcYHeTliywyAyU7%2FGestalt-for-Android%3Fnode-id%3D12157%253A23035%26t%3DKJZySMQdq4Hp0RLc-1" allowFullScreen></iframe>
+
+### Error
+TextArea can display an error message. Don't use errorMessage to provide feedback on character count errors. See the maximum length variant below for more details.
+<iframe style={{border:0}} width="100%" height="300" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FREw1COFYAktmVWrUBh3Ov8%2Fbranch%2FTxlMtK2AcYHeTliywyAyU7%2FGestalt-for-Android%3Fnode-id%3D12157%253A23108%26t%3DKJZySMQdq4Hp0RLc-1" allowFullScreen></iframe>
+
+## Maximum length
+TextArea supports maxLength . Max length sets the maximum number of characters allowed to be entered by the user in TextArea. maxLength must be an integer value 0 or higher.
```

Comment:
Can we use backticks to mark around  `maxLength`, since `maxLength` seems oto be a code prop. That should add highlights

Reply:
I'm not sure that Android uses code props in the same way that web does. We don't have a precedent for it yet. So I adjusted the language to be less formal.